client/src/translations/es.json

```diff
@@ -22,6 +22,7 @@
   "authentication_issue": "Existe un problema verificando su identidad",
   "autodetect_column_types": "Detectar tipos de datos automáticamente",
   "average": "Promedio",
+  "back": "Espalda",
```

Comment:
:smile: sorry but Espalda does mean Back but the body part :smile: here I would say something like "Atrás"

Reply:
:-)